_config.yml

```diff
@@ -0,0 +1,15 @@
+# Site settings
+name: GoDaddy OSS
+email: opensource@godaddy.com
+description: > # this means to ignore newlines until "baseurl:"
```

Comment:
Since there is no `baseurl:` key here, should this comment simply say `until the next key`?